modules/tinymce/CHANGELOG.md

```diff
@@ -15,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Resizing table columns in some scenarios would resize the column to an incorrect position #TINY-7731
 - Image resize backdrop element did not have `data-mce-bogus="all"` set #TINY-7854
 - Resize handles appeared on top of dialogs and menus when using an inline editor #TINY-3263
+- Fixed the `autoresize` plugin incorrectly scrolling to the top of the editor in some cases when changing content #TINY-7291
```

Comment:
```suggestion
- Fixed the `autoresize` plugin incorrectly scrolling to the top of the editor in some cases when changing content #TINY-7291
- Fixed the `editor.selection.scrollIntoView()`  type signature, as it incorrectly required an `Element` instead of `HTMLElement` #TINY-7291
```

@ashfordneil how does that look for a changelog entry, re the types fix?

Reply:
LGTM